DTS-STN/Alpha-Site
# Feature/#128 local nav

# Description

[Local Nav](https://trello.com/c/zlqgf5hp/128-128-local-navigation-no-sign-up-button)

Do a local navigation bar following the Figma design.
![image](https://user-images.githubusercontent.com/72703030/115269719-961a0f80-a109-11eb-9fd0-7cd60115a13b.png)
![image](https://user-images.githubusercontent.com/72703030/115269805-ab8f3980-a109-11eb-990a-4e57a290338a.png)
![image](https://user-images.githubusercontent.com/72703030/115269878-bd70dc80-a109-11eb-89c5-4f5d5c46701d.png)

## Acceptance Criteria

Needs to adjust to different screen sizes and be a dropdown on mobile screens.
Skip the sign-up button

## Test Instructions

1. Test the menu on different screen sizes
2. Dropdown needs to be full screen on mobile
3. Check the hover state

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
